examples/Examples.sln

```diff
@@ -1,7 +1,6 @@
-
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.30501.0
+# Visual Studio 14
```

Comment:
Visual Studio "14" corresponds to Visual Studio 2015. To simplify supporting matrix, i think it is totally fine that we state that VS 2015 and above is required. If that is the direction we are heading to, we just need to update build notes - https://github.com/Microsoft/Mobius/blob/master/notes/windows-instructions.md#building-mobius.